### PR TITLE
feat: General onboarding article for developers want to use KodaDot

### LINKED ISSUE #6462: General onboarding article for developers want to use KodaDot
I guess so for now! 
We will remake it to blog in new form, for general onboarding article for developers

https://kodadot.xyz/first-time/

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/6461#issuecomment-1649835152_

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #6462

image

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

### <samp>🤖 Generated by Copilot at 0889359</samp>

This pull request adds a new blog post to help new contributors get started with KodaDot development, and improves the blog index page layout and functionality. It modifies `pages/blog/index.vue` to fix some UI issues, and creates `content/blog/first-time.md` with the new post content.

### <samp>🤖 Generated by Copilot at 0889359</samp>

> _Sing, O Muse, of the valiant KodaDot contributors_
> _Who, eager to join the NFT revolution, `content` added_
> _A guide for the novices, full of wisdom and skill_
> _How to set up, run, test, and transact with `GraphQL` and `RPC`_

### CODE REVIEW

**Comment by prury** [comment]
i think it needs more spaces between topics:
 
image

font use should follow a magnitude order big -> small -> smaller or at least be the same: 
image

bold? 
image

**Comment by prury** [comment]
@Jarsen136 sorry for the trouble, but I've made some in-browser modifications examples to show what i did to improve readability: 
image
image

basically what i did: spaces between steps and code squares, spaces between h2 titles sections, reduced the links boldness. Capital letters, some text rearrangement for better user understanding.

**Comment by prury** [comment]
> Thanks. I have updated it accordingly. Look like you are really good at writing and article format 😄

thank you so much, yes, i had my share of text reviewing in a distant past 😄.

it is looking really good now, just one last thing and we are good to go (space before MISC): 
image

**Comment by exezbcz** [comment]
helo

I would lower the line height here (h1) 
image
- in the Figma its 56px 


In case of dark mode, I think we can make the heading true white, with paragraph text staying the same. 
image

**Comment by Jarsen136** [comment]
✅ Update

> it is looking really good now, just one last thing and we are good to go (space before MISC):

image

>I would lower the line height here (h1)
> in the Figma its 56px
> In case of dark mode, I think we can make the heading true white, with paragraph text staying the same.

image

**Comment by exezbcz** [comment]
one last thing sir, can we make the code blocks for dark mode too? 
image

**Comment by Jarsen136** [comment]
> one last thing sir, can we make the code blocks for dark mode too? 

Here is it. #6487

**Review by daiagi — APPROVED** [review]
lgtm

File: pages/blog/index.vue [hunk 1]
```diff
@@ -54,7 +54,7 @@ export default {
       const filePath = post.meta.resourcePath
       const fileName = filePath.match(/\/([^/]+)\.\w+$/)[1]
 
-      return fileName
+      return `/blog/${fileName}`
```
Comment (vikiival) [inline pages/blog/index.vue:57]: Why this change?

File: styles/abstracts/_theme.scss [hunk 2]
```diff
@@ -51,7 +51,7 @@ $themes: (
     'k-accentlight': #ffe5f3,
     'k-accentHover': #363234,
     'k-accentlight2': #363234,
-    'k-accentlight2-dark': #ccc,
+    'k-accentlight2-dark': #ffffff,
```
Comment (vikiival) [inline styles/abstracts/_theme.scss:54]: This does not seem right cc @exezbcz

**Review by vikiival — CHANGES_REQUESTED** [review]

File: styles/abstracts/_theme.scss (hunk 2 above)
Comment (exezbcz) [reply styles/abstracts/_theme.scss:54]: `#363234` - this is accentlight2-dark

Comment (vikiival) [reply styles/abstracts/_theme.scss:54]: > `#363234` - this is accentlight2-dark

that is also `k-accentlight2`

so maybe is there some diff?

File: pages/blog/index.vue (hunk 1 above)
Comment (Jarsen136) [reply pages/blog/index.vue]: It's about solving a prefix issue before. It seems not needed now.

**Review by vikiival — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 5.09 USD/DOT ~ 3.929 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0xc32ed363aecebe6b740b4a6e81043a06adcd50f64736e475cbde478f9eb69976](https://polkadot.subscan.io/extrinsic/0xc32ed363aecebe6b740b4a6e81043a06adcd50f64736e475cbde478f9eb69976)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**